src/drafts/MarkdownEditor/_FormattingTools.tsx

```diff
@@ -49,6 +48,10 @@ export const FormattingTools = forwardRef<FormattingTools, {forInputId: string}>
     reference: () => referenceRef.current?.click()
   }))
 
+  useEffect(() => {
+    require('@github/markdown-toolbar-element')
+  })
```

Comment:
```suggestion
  }, [])
```

we don't want to evaluate this every render, just once should be enough